boards/sc/scobc_a1/doc/index.rst

```diff
@@ -21,7 +21,7 @@ Hardware
 Supported Features
 ==================
 
-The Space Cubics OBC module 1 provides the following hardware features:
+The SC-OBC Module A1 provides the following hardware features:
 
 +-----------+------------+------------------------------------+
```

Comment:
n.b. this section should be replaced with the hw-directive block instead (for a future PR)

Reply:
Thanks a lot!  I looked at the other boards and gave it a try in the hw-directive block.